app/config/services.yml

```diff
@@ -20,14 +20,6 @@ services:
     class: AppBundle\Controller\UserController
     autowire: true
 
-  app.controller.adress_controller:
-    class: AppBundle\Controller\AdressController
-    autowire: true
-
-  app.facade.adress_facade:
```

Comment:
No původně jsem chtěl pro adresu vytvořit vlastní kontroler, případně to udělat stejným principem jako jsou produkty a kategorie, ale nakonec jsem to spojil s user kontrolerem. Platí tedy obecně, že pro přehlednost a lepší použitelnost je dobrý mít pro každou entitu vlastní kontroler?